polylogue/local_sync.py

```
@@ -78,6 +81,14 @@ def _sync_sessions(
             html_theme=html_theme,
             **importer_kwargs,
         )
+        if result.skipped:
+            if snapshot is not None:
+                try:
+                    snapshot.unlink()
+                except Exception:
+                    pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)